mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -250,44 +250,40 @@ def _load_mesh(self):  # {{{
             refTopDepth, refLayerThickness
         # }}}
 
-    def _cache_velocity_climatologies(self):  # {{{
+    def _compute_velocity_climatologies(self):  # {{{
         '''compute yearly velocity climatologies and cache them'''
 
-        variableList = ['avgNormalVelocity',
-                        'avgVertVelocityTop']
```

Comment:
You could get the variable names by opening a file with either with `xarray` or with `netCDF4` without actually reading any fields in the data set.  The issue in my mind is more that this is a pain to code up and would clutter up all of our analysis tasks, and I'd want to be sure it's worth it.